components/popover/popover-mixin.js

```diff
@@ -140,6 +145,58 @@ export const PopoverMixin = superclass => class extends superclass {
 				}
 			}
 
+			:host([_mobile][_mobile-tray-location]) .content-width {
+				position: fixed;
+				z-index: 1000;
+			}
+
+			:host([_mobile][_mobile-tray-location="inline-start"]) .content-width,
+			:host([_mobile][_mobile-tray-location="inline-end"]) .content-width {
+				bottom: 0;
+				top: 0;
+			}
+
+			:host([_mobile][_mobile-tray-location="inline-start"]) .content-width {
+				border-bottom-left-radius: 0;
+				border-top-left-radius: 0;
```

Comment:
Would logical properties for the border radiuses and left/right be useful? Curious that RTL isn't broken since there's no RTL-specific styles here...

Reply:
Yeah, but only if switching to support RTL per above comment. RTL wasn't boken per se, since it wasn't support for the tray.